src/unsafe-rust/dereferencing.md

```diff
@@ -52,8 +57,8 @@ In the case of pointer dereferences, this means that the pointers must be
 
 In most cases the pointer must also be properly aligned.
 
-The "NOT SAFE" section gives an example of a common kind of UB bug: `*r1` has
-the `'static` lifetime, so `r3` has type `&'static String`, and thus outlives
-`s`. Creating a reference from a pointer requires _great care_.
+The "UNSOUND" section gives an example of a common kind of UB bug: `*p1` has the
+`'static` lifetime, so `r` has type `&'static i32`, and thus outlives `x`.
+Creating a reference from a pointer requires _great care_.
```

Comment:
`r` isn't outliving `x` anymore -- I think this paragraph needs to be updated

Reply:
Fixed.